addons/test_mail/tests/test_mail_activity.py

```diff
@@ -759,3 +759,24 @@ def test_week_grouping(self):
         self.assertEqual(groups[0][groupby], pg_groups["overdue"])
         self.assertEqual(groups[1][groupby], pg_groups["today"])
         self.assertEqual(groups[2][groupby], pg_groups["planned"])
+
+@tests.tagged('mail_activity')
+class TestActivitySystray(TestActivityCommon):
```

Comment:
Just for the sake of ordering output, could you define it before "TestORM" (and after "TestActivityMixin") ? As class tests are called by alphabetical order, better define them following the same order so that test execution follows the class definitions.